cli/pack/archive.go

```diff
@@ -0,0 +1,66 @@
+package pack
+
+import (
+	"github.com/apex/log"
+	"github.com/tarantool/tt/cli/cmdcontext"
+	"os"
+	"path/filepath"
+)
+
+// archivePacker is a structure that implements Packer interface
+// with specific archive packing behavior.
+type archivePacker struct {
+}
+
+// Run of ArchivePacker packs the bundle into tarball.
+func (packer *archivePacker) Run(cmdCtx *cmdcontext.CmdCtx) error {
+	packCtx := cmdCtx.Pack
+	path, err := pack(cmdCtx)
+	if err != nil {
+		return err
+	}
+	defer func() {
+		err := os.RemoveAll(path)
+		if err != nil {
+			log.Warnf("Failed to remove a temporary directory for packing %s: %s",
+				path, err.Error())
+		}
+	}()
+
+	log.Infof("The passed bundle packed into temporary directory: %s", path)
+
+	var tarName, packageVersion string
```

Comment:
Nit: this would be great to extract getting package version, getting tar name below, to separate functions, which can be tested with unit tests.